libsplinter/src/biome/credentials/store/diesel/mod.rs

```diff
@@ -39,7 +39,7 @@ impl SplinterCredentialsStore {
     /// # Arguments
     ///
     ///  * `connection_pool`: connection pool to the database
-    #[cfg(feature = "biome-rest-api")]
+    #[allow(dead_code)]
```

Comment:
This was correct before -- the guard should just be removed when we remove biome-rest-api feature.